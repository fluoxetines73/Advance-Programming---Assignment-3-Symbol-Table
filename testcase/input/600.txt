RPRINT
BEGIN
RPRINT
BEGIN
END
RPRINT
RPRINT
INSERT f_FF number
INSERT z string
RPRINT
ASSIGN f_FF 456
ASSIGN f_FF 456
BEGIN
LOOKUP z
RPRINT
END
END