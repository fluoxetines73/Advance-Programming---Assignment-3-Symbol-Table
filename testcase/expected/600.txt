



success
success
z//1 f_FF//1
success
success
1
z//1 f_FF//1